src/lib/controllers/aligned-controller.js

```diff
@@ -4,45 +4,9 @@ import NotificationSingleton from '@/lib/notifications/notification-singleton'
 export default class AlignedController {
   /**
    * @param {Vuex Store} store
-   * @param {Object} tokenizeParams - params from application settings
-   *         {String} tokenizer - tokenizer name
-   *         {String} segments - parameter for remote service
    */
-  constructor (store, tokenizeParams = {}) {
+  constructor (store) {

```

Comment:
all methods, that define tokenize workflow steps, are moved to TokenizeController